src/Microsoft.Identity.Client/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -206,7 +217,12 @@ private void RemoveAll(string service)
             {
                 foreach (var record in records)
                 {
-                    SecKeyChain.Remove(record);
+                    var result = SecKeyChain.Remove(record);
+
+                    if (result != SecStatusCode.Success)
+                    {
+                        _requestContext.Logger.Warning("Failed to remove cache record from iOS Keychain: SecStatusCode." + result.ToString());
```

Comment:
Also, remember to include pii logging, as we decided to double the logs for pii and non-pii logs.